c_src/eleveldb.cc

```diff
@@ -274,9 +274,18 @@ ERL_NIF_TERM parse_init_option(ErlNifEnv* env, ERL_NIF_TERM item, EleveldbOption
         else if (option[0] == eleveldb::ATOM_LIMITED_DEVELOPER_MEM)
         {
             if (option[1] == eleveldb::ATOM_TRUE)
+            {
                 opts.m_LimitedDeveloper = true;
+
+                // lower thread count too if developer stuff.
+                //  each thread has 2Mbytes to 8Mbytes of stack
+                if (71==opts.m_EleveldbThreads)
```

Comment:
Do we have this 71 value defined as a macro somewhere? Would be nice to use a manifest constant rather than a hard-coded magic number. Also, do we want to always drop it to 7 if it's greater than 7? That is, should the conditional be "if (7 > opts.m_EleveldbThreads)"? Perhaps not if we want to respect a purposely-set value.